pyproject.toml

```diff
@@ -19,7 +20,8 @@ click = "8.0.3"
 black = {version = "^18.3-alpha.0", allow-prereleases = true}
 pytest = "^3.0"
 pytest-mock = "^1.10"
-moto = "^3.1.6"
+moto = "^3.1.18"
+PyYAML = "^6.0"
```

Comment:
Minor: we may want to increment the version as this is a pretty substantial change

Reply:
Definitely! Thanks! [Update project version to 2.4.0](https://github.com/candidco/confidential/pull/26/commits/da6548f085352700cee28f654cd31a5e44163ea4)